src/deploy.cmd.js

```diff
@@ -176,18 +176,24 @@ class DeployCommand extends AbstractCommand {
 
     const ticks = {};
     const tick = (message, name) => {
+      const shortname = name.replace(/\/package.json.*/, '').replace(/node_modules\//, '');
       bar.tick({
-        action: name ? `packaging ${name}` : '',
+        action: name ? `packaging ${shortname}` : '',
       });
       if (message) {
-        this.log.log({
+        this.log.maybe({
```

Comment:
why?

Reply:
`maybe` only logs when no TTY is attached. I suspected the log message of introducing line breaks.